<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zhuanye</class>
 <widget class="QWidget" name="zhuanye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>专业介绍</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>80</y>
     <width>1141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/image/timg.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/zhuanyejieshao.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="LinkButton_news_zhuye_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 247);
font: 13pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>专业介绍</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="LinkButton_news_zhuye">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 255, 247);
font: 13pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>主页</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;汉仪蝶语体简&quot;;

background-color: rgb(251, 255, 252);</string>
   </property>
   <item>
    <property name="text">
     <string>数据科学与计算机学院（软件学院）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数据科学与计算机学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中国语言文学系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史学系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哲学系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外国语学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>岭南学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>社会学与人类学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>法学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>政治与公共事务管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>资讯管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>传播与设计学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>心理学系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>艺术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>博雅学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>化学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>生命科学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地理科学与规划学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>环境科学与工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>材料科学与工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>逸仙学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电子与信息工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中山医学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>光华口腔医学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公共卫生学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>护理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>药学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中国语言文学系（珠海）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>历史系（珠海）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哲学系（珠海）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>旅游学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>国际关系学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>国际金融学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>国际翻译学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学学院（珠海）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>大气科学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地球科学与工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海洋科学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海洋工程与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>物理与天文学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中法核工程与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>化学工程与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>微电子科学与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>医学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公共卫生学院（深圳）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>药学院（深圳）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>生物医学工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>材料学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电子与通信工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>航空航天学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>智能工程学院</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;院系选择：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;院系所在校区：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>640</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;汉仪蝶语体简&quot;;
background-image: url(:/image/heidi.png);</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;专业设置：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;院系介绍：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;近三年录取分数&lt;/p&gt;&lt;p&gt;(最高、最低、平均)：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;汉仪蝶语体简&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
   <item>
    <property name="text">
     <string>河南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>辽宁</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广东</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>河北</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>山东</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>吉林</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>青海</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>江苏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湖北</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湖南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>浙江</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>江西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>云南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>福建</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>台湾</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>山西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>四川</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>陕西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>贵州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>安徽</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>重庆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>内蒙古</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广西</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西藏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>宁夏</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新疆</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>2018：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>2017：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>2016：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'汉仪蝶语体简';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>691</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;仿宋&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/image/timg.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>580</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_12">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪蝶语体简</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;汉仪蝶语体简&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;科类：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_13">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial&quot;;
background-color: rgb(251, 255, 252);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/cool-background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>label_8</zorder>
  <zorder>LinkButton_news_zhuye_2</zorder>
  <zorder>LinkButton_news_zhuye</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>textBrowser</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>textBrowser_3</zorder>
  <zorder>textBrowser_4</zorder>
  <zorder>textBrowser_5</zorder>
  <zorder>textBrowser_6</zorder>
  <zorder>textBrowser_7</zorder>
  <zorder>textBrowser_8</zorder>
  <zorder>textBrowser_9</zorder>
  <zorder>textBrowser_10</zorder>
  <zorder>textBrowser_11</zorder>
  <zorder>textBrowser_12</zorder>
  <zorder>label_19</zorder>
  <zorder>textBrowser_13</zorder>
  <zorder>label_18</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_11</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>LinkButton_news_zhuye</sender>
   <signal>clicked()</signal>
   <receiver>zhuanye</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
